osxcollector/osxcollector.py

```diff
@@ -355,10 +355,11 @@ def _normalize_val(val, key=None):
     :returns: A string
     """
     # If the key hints this is a timestamp, try to use some popular formats
-    if key and any([-1 != key.lower().find(hint) for hint in ['time', 'utc', 'date', 'accessed']]):
+    if key and any([-1 != key.lower().find(hint) for hint in ['time', 'utc', 'date', 'accessed']]) and not 'times' in key.lower():
```

Comment:
Any ideas if we could refactor the first part of the if statement to also use the `in` keyword instead of `any` + `find` + `in` combo?